src/api/public/apidocs-new/paths/source_project_name_package_name_binary_filename_attribute_name.yaml

```diff
@@ -0,0 +1,70 @@
+get:
+  summary: Get values of an attribute
+  description: |
+    Get attribute and attribute values
```

Comment:
What about mentioning the binary/file here? This definition is maybe too generic :thinking: